src/miner_hbbft_sidecar.erl

```diff
@@ -169,11 +181,12 @@ handle_call({submit, Txn}, From,
                                     case blockchain_txn:absorb(Txn, Chain) of
                                         ok ->
                                             spawn(fun() ->
-                                                      %% this will now return {ok, Position, Length}
-                                                      %% or {error, full} which we could feed back to the caller using gen_server:reply() on the From
-                                                      catch libp2p_group_relcast:handle_command(Group, {txn, Txn})
+                                                        %% this will now return {ok, Position, Length}
+                                                        %% or {error, full} which we could feed back to the caller using gen_server:reply() on the From
+                                                        {ok, QueuePos, QueueLen} = libp2p_group_relcast:handle_command(Group, {txn, Txn}),
```

Comment:
explicit match here seems like it could wedge the calling process since this function is being triggered by a `gen_server:call` with an infinite timeout and this spawned process is not monitored so if this call to the relcast group returns `{error, full}` and crashes on the bad match it'll just go unnoticed and the caller will continue to wait forever. Suggesting a generalized reply in my `query_txn` PR that just wraps either response in a `{_, Height}` tuple like many of the other responses returned from `handle_calls` to the sidecar here.